src/eei.cpp

```diff
@@ -461,6 +461,13 @@ namespace hera {
         dataLength << dec << "\n";
 #endif
 
+      if ((kind == EEICallKind::Call) || (kind == EEICallKind::CallCode)) {
```

Comment:
@lrettig remember why this was only added for call/callcode?

Apparently the logic came from aleth, cc @chfast

Reply:
Only CALL and CALLCODE have value argument.
The depth check should be done for all calls.

Reply:
> Only CALL and CALLCODE have value argument.

Ok so that part is an optimisation.

Reply:
> The depth check should be done for all calls.

Fixed that in #429.